nodejs/10/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM        node:dubnium
+FROM        node:dubnium-jessie
```

Comment:
I'd go with `node:dubnium-buster`, since that contains `apt` v1.8.0 with `apt-transport-https` built-in (https://packages.debian.org/buster/apt)

Reply:
@kuba-kubula neither stretch (debian 9) or buster (debian 10) enable easy installation of mongo 3.4 and lower, the lowest available is 3.6; that's why we decided to stick with jessie for now